modules/metastorage-client/src/main/java/org/apache/ignite/internal/metastorage/client/MetaStorageServiceImpl.java

```diff
@@ -77,111 +78,121 @@ public MetaStorageServiceImpl(RaftGroupService metaStorageRaftGrpSvc) {
     }
 
     /** {@inheritDoc} */
-    @Override public @NotNull CompletableFuture<Entry> get(@NotNull Key key) {
-        return metaStorageRaftGrpSvc.run(new GetCommand(key));
+    @Override public @NotNull CompletableFuture<Entry> get(@NotNull ByteArray key) {
+        return metaStorageRaftGrpSvc.run(new GetCommand(key)).thenApply(obj -> {
```

Comment:
Why not to use 
`.thenApply(MetaStorageServiceImpl::singleEntryResult);`
here?

Reply:
Thx! Fixed.